frontend/apps/reader/modules/readerfont.lua

```diff
@@ -72,6 +72,10 @@ function ReaderFont:setupFaceMenuTable()
     face_list = self:sortFaceList(face_list)
     for k, v in ipairs(face_list) do
         local font_filename, font_faceindex, is_monospace = cre.getFontFaceFilenameAndFaceIndex(v)
+        if not font_filename then
+            -- The font may be available only in italic, for example script fonts
```

Comment:
What do you mean with "script fonts" ?
What would be the use case for a font present only in italic ?